xrdp/xrdp.ini.in

```diff
@@ -196,6 +196,10 @@ ip=127.0.0.1
 port=-1
 #xserverbpp=24
 #delay_ms=2000
+# Disable requested encodings to support buggy VNC servers
+# (1 = ExtendedDesktopSize)
```

Comment:
In config files, we use semicolons to comment out descriptions or supplemental information.

Reply:
This custom is started in #488.